jbmc/regression/jbmc/assume-input-limits/My.java

```diff
@@ -0,0 +1,131 @@
+class My {
+
+  byte byteField;
+  short shortField;
+  int intField;
+  long longField;
+  float floatField;
+  double doubleField;
+
+  int[] intArrField;
+
+  public static void numericalArg(byte byteArg, short shortArg, int intArg, long longArg, float floatArg, double doubleArg) {
+    if (intArg > 3) {
```

Comment:
:pick: put the ifs in the same order as the parameters

Reply:
Done